### PR TITLE
Modify README for public consumption, add release process details. Fixes #87, #141

### LINKED ISSUE #87: Update and expand README.md as appropriate for a public repository
The readme should provide a description of what it is, its purpose and function, etc. We do not need to provide release instructions. It looks like the readme was written before the repo became public.

### PR DESCRIPTION
**Pull request by sa-bpelakh**

### CODE REVIEW

**Review by Jamie-SA — APPROVED** [review]
Probably should include some info about getting it onto the website. Would you prefer creating a separate issue for that or would you like me to commit to your branch?

**Comment by sa-bpelakh** [comment]
@Jamie-SA Feel free to to add to the branch, Rebecca hasn't even reviewed my grammar yet 😄 Since this is a public repo, not sure how detailed we want to be on internal server info, but general instructions are fine.

File: README.md [hunk 1]
```diff
@@ -1,22 +1,10 @@
+![gist logo](gist.png)
 
-# gist upper enterprise ontology
+# gist
```
Comment (rjyounes) [inline README.md:3]: Markdown rules require a space between the header and following text.

**Review by rjyounes — CHANGES_REQUESTED** [review]
I recommend removing the build and release information from the change and release management doc. Otherwise good, including grammar and punctuation :)

File: docs/ChangeAndReleaseManagement.md [hunk 2]
```diff
@@ -49,3 +49,26 @@ Releases
 -----
 
 - Releases occur monthly, approximately on the last working day of the month, unless it is determined during a review meeting that there are not a significant number of substantive changes to warrant a release.
+
```
[the remark this answers is not in the document]
Comment (sa-bpelakh) [reply docs/ChangeAndReleaseManagement.md:52]: Where would you favor documenting this? Confluence? #141 requested the process to be documented.

[the remark this answers is not in the document]
Comment (rjyounes) [reply docs/ChangeAndReleaseManagement.md:52]: I guess I wasn't around for that one. We have documentation of the release processes, both for [developers](https://semarts.atlassian.net/wiki/spaces/OF/pages/49479787/gist+Release+Checklist+Developers) and [ontologists](https://semarts.atlassian.net/wiki/spaces/OF/pages/1126760539/gist+Release+Management+Ontologists), on confluence. Does that work?

File: README.md (hunk 1 above)
Comment (sa-bpelakh) [reply README.md:3]: Done.

File: docs/ChangeAndReleaseManagement.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sa-bpelakh) [reply docs/ChangeAndReleaseManagement.md]: I added a note about attaching the artifact to the release to the [ontologists](https://semarts.atlassian.net/wiki/spaces/OF/pages/1126760539/gist+Release+Management+Ontologists) Confluence page, the rest of the info was already there. Removing it from this public document.

**Review by rjyounes — APPROVED** [review]
Looks good.

**Pull request merged**